packages/rocketchat-autotranslate/server/settings.js

```diff
@@ -1,4 +1,49 @@
 Meteor.startup(function() {
-	RocketChat.settings.add('AutoTranslate_Enabled', false, { type: 'boolean', group: 'Message', section: 'AutoTranslate', public: true });
-	RocketChat.settings.add('AutoTranslate_GoogleAPIKey', '', { type: 'string', group: 'Message', section: 'AutoTranslate', enableQuery: { _id: 'AutoTranslate_Enabled', value: true } });
+	RocketChat.settings.add('AutoTranslate_Enabled', false, {
+		type: 'boolean',
+		group: 'Message',
+		section: 'AutoTranslate',
+		public: true
+	});
+	RocketChat.settings.add('AutoTranslate_ServiceProvider', 'google-translate', {
+		type: 'select',
+		group: 'Message',
+		section: 'AutoTranslate',
+		values: [{
+			key: 'google-translate',
```

Comment:
Since they are all referenced from file inside package `rocketchat-translation`, I'd favor to also add the deepl- and dbs-translation provider into this package. Alternatively, the settings had to be created based upon information received from the registry.